apps/docs/content/components/VideoPlayer.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Video player
+status: Experimental
+source: https://github.com/primer/brand/blob/main/packages/react/src/VideoPlayer/VideoPlayer.tsx
+storybook: '/brand/storybook/?path=/story/components-videoplayer--playground'
+description: Use the video player component to enable playback of self-hosted videos
+---
+
+import {Label} from '@primer/react'
+
+```js
+import {VideoPlayer} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<VideoPlayer title="GitHub media player">
+  <VideoPlayer.Source src="/example.mp4" />
+  <VideoPlayer.Track src="/example.vtt" />
+</VideoPlayer>
+```
+
+## With poster
+
+```jsx live
+<VideoPlayer poster="/example-poster.png" title="GitHub Video Demo">
```

Comment:
Update to title for consistency with previous example

```suggestion
<VideoPlayer poster="/example-poster.png" title="GitHub media player">
```

Reply:
Fixed in the following [SHA](https://github.com/primer/brand/pull/373/commits/ee6ae80b15875236b720a51258e132c66f39e856)